Content.Server/GameObjects/Components/Body/Behavior/LungBehaviorComponent.cs

```diff
@@ -147,6 +149,20 @@ public override void Update(float frameTime)
 
         public override void Inhale(float frameTime)
         {
+            if (Body != null && Body.Owner.TryGetComponent(out InternalsComponent? internals)
+                             && internals.BreathToolEntity != null && internals.GasTankEntity != null
+                             && internals.BreathToolEntity.TryGetComponent(out BreathToolComponent? breathTool)
+                             && breathTool.IsFunctional && internals.GasTankEntity.TryGetComponent(out GasTankComponent? gasTank)
+                             && gasTank.Air != null)
+            {
+                var air = gasTank.RemoveAirVolume(Atmospherics.BreathVolume);
+                if (air == null)
+                    return;
+
+                ToBloodstream(air);
+                return;
```

Comment:
You should pass `air` to the other Inhale method instead, as it handles gas mixtures and this skips the Transfer call to the lungs and goes to the bloodstream directly, which is probably not what you want unless the tank comes with a syringe.

Reply:
tru, made this sane